privacyidea/lib/auditmodules/sqlaudit.py

```diff
@@ -249,7 +253,9 @@ def finalize_log(self):
                           client=self.audit_data.get("client", ""),
                           loglevel=self.audit_data.get("log_level"),
                           clearance_level=self.audit_data.get("clearance_level"),
-                          policies=self.audit_data.get("policies")
+                          policies=self.audit_data.get("policies"),
+                          startdate=self.audit_data.get("startdate"),
+                          duration=duration
```

Comment:
Just a note: these entries won't be included in the audit-log signature.

Reply:
Hm, we have a ``sig_ver`` like "rsa_sha256_pss" but unfortunately we do not now, which data were input for the signature. We might check in ``_log_to_string()``, which is also used during [verify](https://github.com/privacyidea/privacyidea/blob/8a076b7b56b35c298c25b89d1fb044b3eaedba63/privacyidea/lib/auditmodules/sqlaudit.py#L487), if ``startdate`` and ``duration`` is set and only then add this to the string, that is signed and verified. This way we could verify old entries without these values set. 
Do you have a better approach?

Reply:
Done in b97f3d9b

Reply:
Some time ago i did some research on versioning the audit log signature based on the signed values but didn't come to a conclusion. So this works for now.